msa/tb-node/docker/Dockerfile

```diff
@@ -16,12 +16,14 @@
 
 FROM thingsboard/openjdk17:bookworm-slim
 
-COPY start-tb-node.sh ${pkg.name}.deb /tmp/
+COPY logback.xml start-tb-node.sh ${pkg.name}.deb /tmp/
 
 RUN chmod a+x /tmp/*.sh \
     && mv /tmp/start-tb-node.sh /usr/bin && \
     (yes | dpkg -i /tmp/${pkg.name}.deb) && \
     rm /tmp/${pkg.name}.deb && \
+    mv /tmp/logback.xml ${pkg.installFolder}/conf && \
+    chown -R ${pkg.user}:${pkg.user} ${pkg.installFolder}/conf/logback.xml && \
     (systemctl --no-reload disable --now ${pkg.name}.service > /dev/null 2>&1 || :) && \
     chown -R ${pkg.user}:${pkg.user} /tmp && \
```

Comment:
move to first step
```
chown -R ${pkg.user}:${pkg.user} /tmp
```

Reply:
Actually I was going to remove this chown line because I don't understand why do we need it. Do you have any idea?

Reply:
> Actually I was going to remove this chown line because I don't understand why do we need it. Do you have any idea?

it is obsolete I guess. the approach is to copy everything to tmp. set the owner for everything and then move to the right place

I believe the /tmp folder should be writeable for anyone by default from in the source image